pages/utm-builder/[slug].vue

```diff
@@ -29,11 +29,17 @@
 <script setup>
 import UtmGeneratorForm from "@/components/UtmGeneratorForm.vue";
 import Article from "@/components/Article.vue";
+import { types } from "@/data/utmBuilders.js";
 
 const articleType = "utm-builder";
 
 const route = useRoute();
-let { slug, articleSlug } = route.params;
+let { slug } = route.params;
+
+const articleSlug = computed(() => {
+  const article = types.find((v) => v?.type === slug.toLowerCase());
```

Comment:
The variable `v` doesn't say much. I would advise to keep them long (`found`, `type`), or abstract them away like this:

```suggestion
  const article = types.find(({ type }) => type === slug.toLowerCase());
```